src/pages/messages/components/NviApprovals.tsx

```diff
@@ -79,21 +52,46 @@ const InstitutionApprovalStatusRow = ({ approvalStatus }: InstitutionApprovalSta
     gcTime: 1_800_000,
   });
 
-  const institutionAcronym = institutionQuery.data?.acronym ?? '';
+  return (
+    <TableRow>
+      <TableCell>
+        {institutionQuery.isPending ? (
+          <Skeleton />
+        ) : (
+          <Typography>{institutionQuery.data?.acronym ?? getLanguageString(institutionQuery.data?.labels)}</Typography>
+        )}
+      </TableCell>
+      <TableCell>
+        <InstitutionStatus status={approvalStatus.status} />
+      </TableCell>
+      <TableCell align="right">
+        <PublicationPointsTypography sx={{ whiteSpace: 'nowrap' }} points={approvalStatus.points} />
+      </TableCell>
+    </TableRow>
+  );
+};
+
+interface InstitutionStatusProps {
+  status: NviCandidateStatus;
+}
+
+const InstitutionStatus = ({ status }: InstitutionStatusProps) => {
+  const { t } = useTranslation();
+
+  const iconProps: SxProps = { mr: '0.2rem' };
```

Comment:
`iconProps` er lik for hver render, så da kan den legges utenfor sånn at den bare opprettes en gang i stedet for på hver render

```suggestion

const iconProps: SxProps = { mr: '0.2rem' };
  
const InstitutionStatus = ({ status }: InstitutionStatusProps) => {
  const { t } = useTranslation();

```